src/View/Helper/PaginatorHelper.php

```diff
@@ -712,9 +712,10 @@ protected function _modulusNumbers($templater, $params, $options)
             ]);
         }
 
+        $url = array_merge($options['url'], ['page' => $params['page']]);
```

Comment:
This throws undefined indexes
Maybe

    $url = isset($options['url']) ? $options['url'] : [];
    $url = array_merge($url, ['page' => $params['page']]);

I wonder if we could use + operator, though. But that might not work for numeric "passed" attributes.
The array_merge might break with "page" as key, but then again no one should do that anyway.